client/my-sites/people/team-list/team.jsx

```diff
@@ -101,12 +102,12 @@ class Team extends React.Component {
 			people = (
 				<InfiniteList
```

Comment:
It would be awesome if we could replace the very complex `InfiniteList` with something simpler.

One of things that `InfiniteList` does is scroll restoration when navigating back and forth. But that feature should be [built-in](https://react-query.tanstack.com/guides/scroll-restoration) in `react-query` now.